### DISCUSSION TITLE
Allow filtering on empty o2m relationships

### DISCUSSION THREAD

**Discussion by alihassan161820** [directus/directus#12173] [Feature Requests]
I need to figure out a way to retrieve data that don't have relation 

for example : 

if i have relation between school which include students 
i need to filter school which don't include any student

**Comment by alihassan161820** [comment]
Any help ?

**Comment by rijkvanzanten** [comment]
This was just implemented in v9.8! You can now use `?filter[count(students)][_eq]=0` ✨
